vms/vms/templates/vms/base.html

```diff
@@ -9,17 +9,20 @@
     <meta name="author" content="">
     <title>Volunteer Management System</title>
     <!-- Bootstrap core CSS -->
-    <!--<link href="../../static/vms/bootstrap-3.2.0-dist/css/bootstrap.min.css" rel="stylesheet">-->
+    <link href="{% static "vms/bootstrap-3.2.0-dist/css/bootstrap.min.css" %}" rel="stylesheet">
```

Comment:
We need bootstrap-3.2.0-dist/css/bootstrap.min.css for the bootstrap-icons: http://getbootstrap.com/components/

bootstrap-lumen-theme/css/bootstrap.min.css has NO icons